OrbitLinuxTracing/UprobesReturnAddressManager.h

```diff
@@ -94,6 +86,20 @@ class UprobesReturnAddressManager {
       frames_to_patch.push_back(i);
     }
 
+    if (!tid_uprobes_stacks_.contains(tid)) {
+      // There are very rare cases where we don't have uprobes records, but need
+      // to patch some frames. This results either from loosing events or
+      // processing events out of order.
+      if (!frames_to_patch.empty()) {
```

Comment:
Thinking about it, this is the same case as `if (num_unique_uprobes < frames_to_patch.size())`, just that `num_unique_uprobes` is zero. Would it make sense to join the two? Or at least let's make clear that it's the same case.

Reply:
yes, I can try to join the two cases.

Reply:
If the code gets worse I guess it's fine to just comment properly to show the parallel between the two.

Reply:
It actually did, as in order to compute the number of unique uprobes, I obviously need the uprobes. 
I decided to make the comments more similar and refer to the first one in the second one.